docs/components.md

```diff
@@ -80,3 +83,38 @@ or some custom style, i.e.
   error
 </Alert>
 ```
+
+Note: in order to let React Dev Tools show the component correctly (instead of an ugly `<Unknown/>`) it is important to
+**not** export and declare the component in the same statement.
+
+#### Don't
+
+```tsx
+export const Alert: FunctionComponent<IAlert> = props => {}
+```
+
+#### Do
+
+```tsx
+const Alert: FunctionComponent<IAlert> = props => {
+
+export { Alert };
+```
+
+In case of a `React.forwardRef`, make sure to use a named function and to set the component's display name
+
+
+#### Don't
+
+```tsx
+export const BlockQuote: React.FunctionComponent<IBlockQuote> = React.forwardRef<{}>((props, ref) => {{}}
+```
+
+#### Do
+
+```tsx
+const BlockQuote: React.FunctionComponent<IBlockQuote> = React.forwardRef<{}>(function BlockQuote(props, ref) {{}}
```

Comment:
We don't really need both.

Reply:
That's right, we can probably get rid of the named function (although it's a nice fallback in case you forgot to set the `displayProperty`)